include/git2/submodule.h

```diff
@@ -108,6 +108,18 @@ typedef enum {
 	GIT_SUBMODULE_STATUS_WD_UNTRACKED)) != 0)
 
 /**
+ * Function pointer to receive each submodule
+ *
+ * `sm` is the `git_submodule` currently being visited.
+ *
+ * `name` is the name of the submodule.
+ *
+ * `payload` is the value you passed to the foreach function as payload.
```

Comment:
These should be in `@param sm <explanation>` format so it shows up pretty in the docs